docs/reference/commandline/service_update.md

```diff
@@ -26,6 +26,12 @@ Options:
       --constraint-rm value              Remove a constraint (default [])
       --container-label-add value        Add or update container labels (default [])
       --container-label-rm value         Remove a container label by its key (default [])
+      --dns-add list                     Add or update custom DNS servers (default [])
```

Comment:
DNS options of a running container can't be modified. We should provide the dns options only on service create.